src/region/RegionCreator.tsx

```diff
@@ -175,12 +188,14 @@ export default function RegionCreator({ className, onAbort, onStart, onStop }: P
         const { current: regionRect } = regionRectRef;
         const shape = getShape();
 
-        if (isDirty && regionRect && shape) {
+        if (isDirty && regionRect) {
```

Comment:
By removing `shape` that means we are restricting a user from actually creating a small region, right?